Mathlib/MeasureTheory/Measure/Lebesgue/Integral.lean

```diff
@@ -74,6 +75,15 @@ mathlib's conventions for integrals over finite intervals (see `intervalIntegral
 of finite integrals, see `intervalIntegral.integral_comp_neg`.
 -/
 
+theorem MeasureTheory.IntegrableOn.comp_neg_Iic {E : Type*} [NormedAddCommGroup E]
```

Comment:
Presumably this and the following result generalize to the setting where the domain is a linearly ordered additive group, yes? Please generalize to that situation.